Source/CarthageKit/GitHub.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 import Result
 import ReactiveCocoa
 import Runes
+import Box
```

Comment:
Sorry for that I forgot to delete it. I was trying a different approch that didn't work and I left that. I am adding a new commit to fix it.